festim/mesh/mesh.py

```diff
@@ -86,10 +88,29 @@ def define_meshtags(self, surface_subdomains, volume_subdomains):
             entities = vol.locate_subdomain_entities(self.mesh, self.vdim)
             tags_volumes[entities] = vol.id
 
-        # define mesh tags
-        facet_meshtags = meshtags(self.mesh, self.fdim, mesh_facet_indices, tags_facets)
         volume_meshtags = meshtags(
             self.mesh, self.vdim, mesh_cell_indices, tags_volumes
         )
 
+        # tag interfaces
+        interfaces = interfaces or {}  # if interfaces is None, set it to empty dict
+        for interface in interfaces:
+            (domain_1, domain_2) = interface.subdomains
+            all_1_facets = dolfinx.mesh.compute_incident_entities(
+                self.mesh.topology,
+                volume_meshtags.find(domain_1.id),
+                self.vdim,
+                self.fdim,
+            )
+            all_2_facets = dolfinx.mesh.compute_incident_entities(
+                self.mesh.topology,
+                volume_meshtags.find(domain_2.id),
+                self.vdim,
+                self.fdim,
+            )
+            interface_entities = np.intersect1d(all_1_facets, all_2_facets)
+            tags_facets[interface_entities] = interface.id
+
+        facet_meshtags = meshtags(self.mesh, self.fdim, mesh_facet_indices, tags_facets)
+
```

Comment:
The reason why these lines are not covered is because the only multi-material case has meshtags provided by the user so `define_meshtags` isn't called at all.

We would need a 1D test case